src/interfaces/token.ts

```diff
@@ -100,6 +100,18 @@ export class TokenInterface<C extends ChainId> extends ServiceInterface<C> {
         const icon = icons[token.address];
         return icon ? { ...token, icon } : token;
       });
+    } else if (this.chainId === 250) {
+      const price = await this.yearn.services.oracle.getPriceUsdc(WftmAddress);
+      const ftm: Token = {
+        address: ZeroAddress,
+        icon: this.yearn.services.asset.icon(ZeroAddress),
+        priceUsdc: price,
+        name: "Fantom",
+        symbol: "FTM",
+        decimals: "18",
+        supported: { wftm: true }
+      };
```

Comment:
Since we will be adding more tokens later, I would prefer we fetch most of this data from yearn-meta. We can then limit the hardcode to just addresses.

Reply:
done, but have had to add the hardcoding for ftm